ui/app/components/ui/tooltip.js

```diff
@@ -1,6 +1,175 @@
 import PropTypes from 'prop-types'
-import React from 'react'
-import ReactTooltip from 'react-tooltip-component'
+import React, {Component} from 'react'
+import ReactDOM from 'react-dom'
+
+// TODO: find a suitable replacement for this
```

Comment:
Can we create (or find?) an issue for this work and then drop this comment?

Reply:
I'm cool to create an issue, but think there's also value in leaving a `// TODO:` as well?